FitGoal/FitGoal/UserProfileConfiguration/ViewControllers/UserProfileConfiguratorViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  UserGenderViewController.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 26/4/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class UserProfileConfiguratorViewController: UIViewController {
+    
+    private let selectorView: UIView
+    private var selectorViewHeight: CGFloat?
+    
+    private let questionPrefix = UILabel()
+    private let questionSuffix = UILabel()
+
+    init(selectorView: UIView, questionPrefix: String, questionSuffix: String) {
+        self.selectorView = selectorView
+        super.init(nibName: nil, bundle: nil)
+        self.selectorViewHeight = calculateViewHeight(for: self.selectorView)
+        configureQuestions(prefix: questionPrefix, suffix: questionSuffix)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureQuestions(prefix: String, suffix: String) {
+        questionPrefix.attributedText = prefix.formattedText(font: "Roboto-Light", size: 19, color: .white, kern: 0.23)
+        questionSuffix.attributedText = suffix.formattedText(font: "Oswald-Medium", size: 50, color: .white, kern: 0.59)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let views = [
+            selectorView,
+            questionPrefix,
+            questionSuffix
+        ]
+        
+        view.addMultipleSubviews(views)
+        setConstraints()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        guard let height = selectorViewHeight else {
+            assertionFailure("SelectorView is incorrect")
+            return
+        }
+        
+        selectorView.frame = CGRect(x: 0, y: 0, width: 320, height: height)
+        selectorView.center = CGPoint(x: view.center.x, y: view.center.y + 30)
+    }
+    
+    private func calculateViewHeight(for view: UIView) -> CGFloat? {
+        if view is FitnessLevelChooserView {
+            return 320
+        } else if view is GenderView {
+            return 320
+        }
+        return nil
+    }
```

Comment:
What's this function for? It looks kind of weird that it always returns 320 or `nil`. What do you use the `nil` return value for?